src/app/components/card-hero/card-hero.component.ts

```diff
@@ -29,12 +29,10 @@ export class CardHeroComponent implements OnInit {
   }
 
   ngOnInit(): void {
-    const id = +this.route.snapshot.paramMap.get('id');
-
+    const id = Number.parseInt(this.route.snapshot.paramMap.get('id'));
```

Comment:
`Number.parseInt` is too heavy, use just parseInt and set radix 10 `parseInt('123', 10)`